docs/source/examples/cirq-ibmq-backends.myst

```diff
@@ -0,0 +1,264 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(label-cirq-zne-example)=
+# Error mitigation on IBMQ backends with Cirq
```

Comment:
```suggestion
# Error mitigation with Cirq on IBMQ backends
```
I think swapping the order makes a little bit more sense as we expect these "translations" to beneficial to, say Cirq users.

If we want to make this change it will need to be changed in a few other locations as well.